libs/custom-paper-handler/src/driver/coders.ts

```diff
@@ -137,6 +137,41 @@ export type RealTimeExchangeResponseWithoutData = CoderType<
   typeof RealTimeExchangeResponseWithoutData
 >;
 
+export const RealTimeStatusTransmission = message({
```

Comment:
In what way is this different from the `PrinterStatusRealTimeExchangeResponse` above?

Reply:
It's the same but lacks the real-time response header. I can factor it out and use spread syntax.

Reply:
Your call, just wanted to make sure the code duplication was intentional.

Reply:
Good call, [5802d37](https://github.com/votingworks/vxsuite/pull/4846/commits/5802d37fbb73a3e26c00135302c8f4a12fc44a6e)